packages/bbui/src/FancyForm/FancyField.svelte

```diff
@@ -91,8 +101,8 @@
   }
   .content {
     position: relative;
-    height: 64px;
-    padding: 0 16px;
+    height: var(--fancy-field-height);
```

Comment:
Same here, this can break any current usage

Reply:
Also has a default option set in the variable